lib/rspec/support/spec/library_wide_checks.rb

```diff
@@ -68,8 +68,13 @@ def load_all_files(files, preamble, postamble=nil)
       run_ruby_with_current_load_path(command, *options)
     end
 
-    # Ignore bundler warning.
-    stderr = stderr.split("\n").reject { |l| l =~ %r{bundler/source/rubygems} }.join("\n")
+    stderr = stderr.split("\n").reject do |l|
+      # Ignore bundler warning.
+      l =~ %r{bundler/(source/)?rubygems} ||
+      # Ignore bundler + rubygems warning.
+      l =~ %r{site_ruby/\d\.\d\.\d/rubygems} ||
+      l =~ %r{lib/bundler/rubygems}
```

Comment:
What's an example of a string that matches this but not the first?

Sent from my iPhone

> On May 25, 2016, at 6:02 PM, Jon Rowe <notifications@github.com> wrote:
> 
> In lib/rspec/support/spec/library_wide_checks.rb:
> 
> > @@ -68,8 +68,13 @@ def load_all_files(files, preamble, postamble=nil)
> >        run_ruby_with_current_load_path(command, *options)
> >      end
> >  
> > -    # Ignore bundler warning.
> > -    stderr = stderr.split("\n").reject { |l| l =~ %r{bundler/source/rubygems} }.join("\n")
> > +    stderr = stderr.split("\n").reject do |l|
> > +      # Ignore bundler warning.
> > +      l =~ %r{bundler/(source/)?rubygems} ||
> > +      # Ignore bundler + rubygems warning.
> > +      l =~ %r{site_ruby/\d\.\d\.\d/rubygems} ||
> > +      l =~ %r{lib/bundler/rubygems}
> I'm not sure why the first regex doesn't work, but it doesn't :/
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
>